### PR TITLE
Update apk.py

### PR DESCRIPTION
**Pull request by ranjeetDT**
solution to the issue :
https://github.com/androguard/androguard/issues/148

### CODE REVIEW

File: androguard/core/bytecodes/apk.py [hunk 1]
```diff
@@ -182,6 +183,7 @@ def __init__(self,
             self.__raw = read(filename)
 
         self.zipmodule = zipmodule
+        print zipmodule
```
Comment (subho007) [inline androguard/core/bytecodes/apk.py]: Remove this print @ranjeetDT

Comment (ranjeetDT) [inline androguard/core/bytecodes/apk.py]: removed

**Comment by subho007** [comment]
@adesnos I think you can merge this, it looks good :+1:

**Comment by reox** [comment]
Can you try to merge it with the new code changes?

**Comment by reox** [comment]
By the way, is this still an issue after the py2/3 changes?

**Comment by subho007** [comment]
Closing this PR since the latest code in androguard python works great.

**Pull request closed without merging**